README.md

```diff
@@ -36,581 +34,25 @@ The following set of features makes Artipie unique among all others:
-Then to push your Docker image use the following command:
-
-```bash
-$ docker push my-docker.my-company.com/my-image
-```
-
-To pull the image use the following command:
-
-```bash
-$ docker pull my-docker.my-company.com/my-image
-```
-
-### Docker Proxy Repo
-
-Try this `docker-proxy.yaml` file to host a proxy to `mcr.microsoft.com` registry:
-
-```yaml
-repo:
-  type: docker-proxy
-  settings:
-    host: mcr.microsoft.com
-```
-
-Artipie will redirect all pull requests to specified registry.
-
-Proxy repository supports caching in local storage.
-To enable it and make previously accessed images available when source repository is down 
-add `storage` section to config:
-
-```yaml
-repo:
-  type: docker-proxy
-  settings:
-    host: mcr.microsoft.com
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-docker-cache
-```
-
-### Python repo
-
-Try this `pypi.yaml` file:
-
-```yaml
-repo:
-  type: pypi
-  storage:
-    type: fs
-    path: /tmp/artipie/data/python-repo
-```
-
-Publish a package(whl or tar.gz):
-  * Install twine utility, if you don't do it already [docs](https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives).
-```bash
-$ python3 -m pip install --user --upgrade twine
-```
-  * build the package, as described in python docs
-  * upload to server with a command
-```bash
-$ python3 -m twine upload --repository-url http://localhost:8080/pypi/ -u user.name -p pass testpkg/dist/*
-```
-
-Install a package:
-
-```bash
-$ pip install --index-url http://localhost:8080/pypi/ testpkg
-```
-
-## Additional configuration
-
-Environment variables:
- - `SSL_TRUSTALL` - trust all unkown certificates
-
-## Multitenancy
-
-You may want to run Artipie for your company, which has a few teams.
-Each team may want to have its own repository. To do this, you create
-a global configuration file `/etc/artipie.yml`:
-
-```yaml
-meta:
-  layout: org
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-docker
-  credentials:
-    type: file
-    path: _credentials.yml
-```
-
-If the `type` is set to `file`, another YAML file is required in the storage, with
-a list of users who will be allowed to create repos
-(each `pass` is combination or either `plain` or `sha256` and a text):
-
-
-```yaml
-credentials:
-  jane:
-    pass: "plain:qwerty"
-  john:
-    pass: "sha256:xxxxxxxxxxxxxxxxxxxxxxx"
-```
-
-If the `type` is set to `env`, the following environment variables are expected:
-`ARTIPIE_USER_NAME` and `ARTIPIE_USER_PASS`. For example, you start
-Docker container with the `-e` option:
-
-```bash
-docker run -d -v /var/artipie:/var/artipie` -p 80:80 \
-  -e ARTIPIE_USER_NAME=artipie -e ARTIPIE_USER_PASS=qwerty \
-  artipie/artipie:latest
-```
-
-## Metrics
-
-You may enable some basic metrics collecting and periodic publishing to application log
-by adding `metrics` to `meta` section of global configuration file `/etc/artipie.yml`:
-
-```yaml
-meta:
-  metrics:
-    type: log
-```
-
-## How to contribute
```

Comment:
@Sammers21 I'd suggest to leave "How to contribute" in the bottom of the main readme

Reply:
@olenagerasimova, the contribution policy will be at CONTRIBUTING.md after #308 merge